openslides/users/apps.py

```diff
@@ -29,5 +29,5 @@ def ready(self):
         register_slide_model(User, 'participant/user_slide.html')
 
         # Register viewsets.
-        router.register('users/user', UserViewSet)
-        router.register('users/group', GroupViewSet)
+        router.register('user/user', UserViewSet)
```

Comment:
I am sure. We thought whether to rename the participant app to 'user' or 'users'. @DebVortex advised the plural form so you chose it. Of course we do not rename 'core' to 'cores' for we only have one system core. And 'agenda' is already a (latin) plural form.